### ISSUE TITLE
Visual regression in base_show_macro.html.twig

### ISSUE THREAD

**Issue by soerenbernstein** [sonata-project/SonataAdminBundle#3738]
| Question       | Answer
|----------------|-------------------------------------------------------------
| Bundle version | dev-master 79153b8
| Symfony version| v2.7.11
| php version    | 5.6.18

# Error message

```
The show view template will not enlarge the conent area if the page is higher then the browser window.
```

# Steps to reproduce
Use an admin show thats higher than the browser window

# Expected results
![sonata admin expected](https://cloud.githubusercontent.com/assets/3931047/14733186/7d042846-0861-11e6-8654-6a3e86415346.png)

# Actual results
![sonata admin actual](https://cloud.githubusercontent.com/assets/3931047/14733192/7f5e559e-0861-11e6-984b-20751e80755a.png)

# Probable solution
Insert the `<div class="row">` also for non-tabbed views

**Comment by ju1ius** [comment]
@soerenbernstein Fixed in b2c1a206ad3384ffcbc1b03491ae5756e6ad6e61

**Comment by OskarStark** [comment]
thanks for your feedback @ju1ius
